### PR TITLE
add locales

### PR DESCRIPTION
**Pull request by mariobodemann**
This PR will add the `/locales` endpoint.

### CODE REVIEW

File: src/main/java/com/contentful/java/cma/ModuleLocales.java [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+    }
+  }
+
+  /**
+   * Update the given locale instance on Contentful.
+   * <p>
+   * Please make sure that the instance provided is fetched from Contentful. Otherwise you will
+   * get an exception thrown.
+   *
+   * @param locale the locale fetched from contentful, updated by caller, to be updated.
```
Comment (dborsatto) [inline src/main/java/com/contentful/java/cma/ModuleLocales.java]: There are some instances, like here, where _Contentful_ is not capitalized, as it should be 🙂

File: src/main/java/com/contentful/java/cma/model/CMALocale.java [hunk 2]
```diff
@@ -182,11 +185,49 @@ public CMALocale setDefault(boolean isDefault) {
...
+  }
+
+  /**
    * @return a human readable string, representing the object.
    */
   @Override public String toString() {
     return "CMALocale { " + super.toString() + " "
         + "code = " + getCode() + ", "
+        + "contentDeliveryApi = " + isContentDeliveryApi() + ", "
+        + "contentManagementApi = " + isContentManagementApi() + ", "
```
Comment (dborsatto) [inline src/main/java/com/contentful/java/cma/model/CMALocale.java]: Are boolean automatically cast to string, in Java? 🙂

**Comment by mariobodemann** [comment]
@dborsatto Love the feedback! Will incorporate it asap ... :D

**Comment by dborsatto** [comment]
Code is fine, it's just small details that you may want to take a look at before merging ;)

File: src/main/java/com/contentful/java/cma/model/CMALocale.java (hunk 2 above)
Comment (mariobodemann) [reply src/main/java/com/contentful/java/cma/model/CMALocale.java]: Yes, almost. Actually they are first boxed to `Boolean` and then it's `.toString()` is called ... ;)

**Pull request merged**